hack/kops-patch.yaml

```diff
@@ -172,3 +172,6 @@ spec:
       ExpandCSIVolumes: "true"
       CSIInlineVolume: "true"
```

Comment:
BTW when bumping K8S_VERSION we can remove CSIDriverRegistry, CSINodeInfo, and CSIBlockVolume, all of these are GA since 1.18 so no need to explicitly set true

Reply:
ack, will do 👍